utils/kata-manager.sh

```diff
@@ -669,16 +669,26 @@ handle_kata()
 	kata-runtime --version
 }
 
+containerd_version_number()
+{
+	local flavour="${1:-lts}"
+
+	base_version="$(curl -fsSL https://raw.githubusercontent.com/containerd/containerd.io/main/content/releases.md | grep -iE "\[d+.d+\].*| $flavour" | grep -oE "\[[0-9]+.[0-9]+\]" | grep -oE "[0-9]+.[0-9]+")"
+
```

Comment:
Could you fold this long line for clarity?

```suggestion
	base_version="$(curl -fsSL https://raw.githubusercontent.com/containerd/containerd.io/main/content/releases.md |\
	grep -iE "\[d+.d+\].*| $flavour" |\
	grep -oE "\[[0-9]+.[0-9]+\]" |\
	grep -oE "[0-9]+.[0-9]+")"

```

Reply:
Done, thanks for the suggestion!